content/en/docs/reference/command-line-tools-reference/feature-gates.md

```diff
@@ -69,7 +69,8 @@ different Kubernetes components.
 | `CSIMigrationGCE` | `false` | Alpha | 1.14 | |
 | `CSIMigrationOpenStack` | `false` | Alpha | 1.14 | |
 | `CSINodeInfo` | `false` | Alpha | 1.12 | 1.13 |
-| `CSINodeInfo` | `true` | Beta | 1.14 | |
+| `CSINodeInfo` | `true` | Beta | 1.14 | 1.16 |
+| `CSINodeInfo` | `true` | GA | 1.17 | |
```

Comment:
Should be the whole feature moved to the "Feature gates for graduated or deprecated features" table below?

Reply:
done